Backpack/src/main/java/net/skyscanner/backpack/calendar/view/CalendarView.kt

```diff
@@ -105,5 +111,7 @@ internal class CalendarView constructor(
     val currScroll = (view.firstVisiblePosition * child.height - child.bottom).toLong()
     previousScrollPosition = currScroll
     previousScrollState = currentScrollState
+
```

Comment:
I believe the TODO at the top of the `onScroll` is not relevant anymore -- it was there as a reminder until the pill is released

Reply:
removed overridden method